lib/_stream_readable.js

```diff
@@ -1139,3 +1139,42 @@ function endReadableNT(state, stream) {
     }
   }
 }
+
+Readable.from = function(iterable, opts) {
+  let iterator;
+  if (iterable && iterable[Symbol.asyncIterator])
+    iterator = iterable[Symbol.asyncIterator]();
+  else if (iterable && iterable[Symbol.iterator])
+    iterator = iterable[Symbol.iterator]();
+  else
+    throw new ERR_INVALID_ARG_TYPE('iterable', ['Iterable'], iterable);
+
+  const readable = new Readable({
+    objectMode: 'objectMode' in opts ? opts.objectMode : true,
```

Comment:
```suggestion
    objectMode: true,
```

The `in` operator throws if `opts` is `undefined` and if opts has an `objectMode` property, whatever we set here will be overwritten in the spread.

Reply:
@guybedford sorry if I didn't explain clearly. My suggestion was:

```js
new Readable({
  objectMode: true,
  ...opts
})
```

Reply:
Ah, yes that is much nicer thanks.